### ISSUE TITLE
Invalid block miner resulting in payFees failing

### ISSUE THREAD

**Issue by Sriep** [0chain/0chain#2240]
MinerBlockRewards System test has identified a situation where [payFees](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/minersc/fees.go#L236) sometimes fails.

The problem is that the block miner is invalid; this results in the endpoint failing with [trying to access this miner. ](https://github.com/0chain/0chain/blob/staging/code/go/0chain.net/smartcontract/minersc/fees.go#L277), with an error like 
```
pay_fee: can't get generator 'c6f4b8ce5da386b278ba8c4e6cf98b24b32d15bc675b4d12c95e082079c91937': value not present
```

The cause of this issue appears to be the block miner selection process selecting an invalid miner.

**Comment by peterlimg** [comment]
@Sriep do you see the error of `common.NewError("pay_fee", "jumped back in time?")` as you linked in the `trying to access this miner` above? or only see the `can't get generator ...: value not present` error. If we saw the `jumped back in time` error, we might re-entry the round's block state computation, i.e we could have computed block `b1` in round `100` for example, and we try to compute the state of a block in round `100` again later with the computed state of `b1`, which would cause big troubles.

**Comment by peterlimg** [comment]
@Sriep can this be closed via #2260

**Comment by peterlimg** [comment]
Should have been fixed by #2260, feel free to re-open if not
